packages/storybook/stories/Examples/TeamsInterop/TeamsInteropDocs.tsx

```diff
@@ -32,12 +32,13 @@ export const getDocs: () => JSX.Element = () => {
       <Source code={LobbyComponentText} />
       <Heading>Inline Images</Heading>
       <SingleLineBetaBanner />
+      <Description>Microsoft Teams allows users to share images by:</Description>
+      <Description>- Copying & paste into the box at the bottom of the chat.</Description>
+      <Description>- Drag & drop into the chat area.</Description>
```

Comment:
Change this to "Drag & drop into the input box area of the chat"?

Reply:
i am not sure if input box is too technical, but the wording is from PR review team